static_src/components/user_role_list_control.jsx

```diff
@@ -82,7 +82,12 @@ export default class UserRoleListControl extends React.Component {
   }
 
   roles() {
-    const roles = this.props.user.roles;
+    let roles;
+    if (this.props.userType == 'space_users') {
```

Comment:
going to move this `'space_users'` into a constant in the next PR because it's pretty widely used and don't want to create it in this PR and cause confusion.